DC/5.6.0/Searches/GetDownloadQualities.dcl

```diff
@@ -0,0 +1,39 @@
+resource search get_download_qualities {
+    version_id = resource.product.digizuite_dam_center.base_version_id
+    name = 'GetDownloadQualities'
+    is_customizable = false
+    search_xml = '<search name="GetDownloadQualities">
+  <searchSection>
+    <searchFields>
+      <searchField id="criMemberGroupId" fieldName="member_group.member_groupid" operator="AND" renderType="string" valueHandler="Equals" visible="true" />
+      <searchField id="criMetaFieldGroupId" fieldName="member_group.item_metafield_groupid" operator="AND" renderType="string" valueHandler="Equals" visible="false">
+        <values valueType="String">
+          <value>10044</value>
+        </values>
+      </searchField>
+      <searchField id="language" type="language" valueHandler="Equals" visible="false">
+        <values valueType="String">
+          <value>3</value>
+        </values>
+	  </searchField>
+      <searchField id="security" operator="AND" securityType="ItemSecurityReadAccess" visible="false" />
+    </searchFields>
```

Comment:
I think the values here should be DCL references to the appropriate resources - e.g. the English language.